### ISSUE TITLE
Questions about usage

### ISSUE THREAD

**Issue by frios** [MojtabaHs/iPhoneNumberField#21]
Thank you for producing this component. I am finding it very useful, but I have a few issues that I hope you can help me resolve.

I have a “page-wide” disabled state that disallows any text field from receiving input unless a specific button is pressed. So upon display of the page, if the user taps on a text field, nothing happens. I do this by using the following code for each textfield on my page:
```
.disabled(editing ? false : true)
```
When I use your iPhoneNumberField, I notice a few things. 

1) When I first display the page, the text color is gray. I would prefer for it to be standard label color, so I used the modifiers

 ```               
                .foregroundColor(.label)
                .placeholderColor(.label)
```
but they did not have any initial impact. If I navigate away from the page and them come back to it, the text is the correct label color.

2) When i tap on the iPhoneNumberField it still gets input (and brings up a number keyboard) even though the field is supposed to be disabled. 

3) if I use the isEditing parameter and set it to my page-wide editing var then when the iPhoneNumberField is tapped, the state of the whole page editing is changed (which I understand)

So the questions I have are as follows:

1) How can I totally disable the iPhoneNumberField so that it does not allow any input when my editing var is false?

2) How can I set the text color to always show up in the standard label color?

Thank you for your help on these issues.

**Comment by MichealSBingham** [comment]
I have the same issue, the `.disabled()` modifier does not seem to work, have you found a solution?

**Comment by frios** [comment]
Honestly, I just stopped using it.

**Comment by MojtabaHs** [comment]
1. `disabled` modifier fixed in v0.6.2
2. Unfortunately, Apple does not allow accessing the `foregroundColor`. But you can always customize almost anything in the field by accessing directly the underlying `UIView`:

```
            iPhoneNumberField(
                "Title",
                text: $text,
                isEditing: $isEditing,
                formatted: true) {
                    $0.numberPlaceholderColor = .red
                    $0.textColor = .label // <- Here
                    $0.numberPlaceholderColor = .label // <- Here
                }
```
